README.md

```diff
@@ -22,46 +22,51 @@ Additional description about the project and its features.
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+**Download code from my repositor and run local in your computer.**
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
 ### Setup
 
+open my GitHub download code
+
 ### Install
 
+install any code editor (vs code or atom)
```

Comment:
- [ ] As mentioned by the previous reviewer, it seems that you did not provide useful instructions on how to set up and run the project locally, remember you are using webpack, this means we need to install all the dependencies and know which command to run if we need to start the app locally but for now you did not include this information,  can you please add it? 😉